src/Form.tsx

```diff
@@ -146,13 +147,19 @@ const Form: React.ForwardRefRenderFunction<FormInstance, FormProps> = (
 
   return (
     <Component
-      {...restProps}
       onSubmit={(event: React.FormEvent<HTMLFormElement>) => {
         event.preventDefault();
         event.stopPropagation();
 
         formInstance.submit();
       }}
+      onReset={(event: React.FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        event.stopPropagation();
+
+        formInstance.resetFields();
```

Comment:
restProps.onReset();

Reply:
`preventDefault` is good enough. Should not `stopPropagation` .

Reply:
OK, thanks.